topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -293,131 +293,3 @@
     - Cluster queue depth?
     - ...?
 - Can dynamically modify destinations in job config (i.e. `sbatch` params)
-
----
-
-# Galaxy Dependency Resolver Configuration
-
-[Full documentation](https://docs.galaxyproject.org/en/master/admin/dependency_resolvers.html)
-
----
-
-# Dependency resolvers
-
-Specifies the order in which to attempt to resolve tool dependencies, e.g.:
-```xml
-<dependency_resolvers>
-    <tool_shed_packages />
-    <galaxy_packages />
-    <conda />
-    <galaxy_packages versionless="true" />
-    <conda versionless="true" />
-    <modules modulecmd="/opt/Modules/3.2.9/bin/modulecmd" />
-    <modules modulecmd="/opt/Modules/3.2.9/bin/modulecmd" versionless="true" default_indicator="default" />
```

Comment:
1 slide saying "Galaxy can use Modules, consult whatever slides the dependency resolver config ends up in" might be reasonable?

Reply:
yeah good idea!